### PR TITLE
Vector Icons

### PR DESCRIPTION
**Pull request by tatarize**
Switch over some icons for vector icons.

Scale better.

Use paid icon8 license.

### CODE REVIEW

**Comment by tatarize** [comment]
Vector icons color doesn't change on disable.

**Comment by jpirnay** [comment]
> Vector icons color doesn't change on disable.

Fixed

File: meerk40t/gui/preferences.py [hunk 1]
```diff
@@ -525,36 +526,47 @@ def __init__(self, *args, **kwds):
...
 
         _icon = wx.NullIcon
         _icon.CopyFromBitmap(icons8_administrative_tools_50.GetBitmap())
         self.SetIcon(_icon)
         self.SetTitle(_("Preferences"))
 
+    @signal_listener("preferences")
+    def on_pref_signal(self, origin, *args):
+        if len(args):
+            panel = args[0]
```
Comment (tatarize) [inline meerk40t/gui/preferences.py:565]: Should abort if the args aren't there.

**Comment by tatarize** [comment]
Waiting on @mjalindahl's vector icons. But, most of the rest of this should be merged, it has a couple unrelated PR that are going kind of stale.

**Pull request merged**